apps/condo/domains/banking/utils/serverSchema/converters/convertFrom1CExchangeToSchema.js

```diff
@@ -225,12 +221,19 @@ function initNode (line) {
                     },
                 }
                 if (isOutcome) {
-                    transactionData.contractorAccount = {
-                        name: values['Получатель1'],
-                        tin: values['ПолучательИНН'],
-                        number: values['ПолучательРасчСчет'],
-                        bankName: values['ПолучательБанк1'],
-                        routingNumber: values['ПолучательБИК'],
+                    const name = values['Получатель1'] || values['Получатель']
+                    const number = values['ПолучательРасчСчет']
+                    const bankName = values['ПолучательБанк1']
+                    const routingNumber = values['ПолучательБИК']
+                    const tin = values['ПолучательИНН']
+                    if (name && number && bankName && routingNumber && tin) {
```

Comment:
To avoid loosing data, maybe to set a placeholder instead of name with value "[Unknown]"